app/src/js/components/ReconciliationReports/reconciliation-report.js

```diff
@@ -49,25 +36,17 @@ const parseFileObject = (d) => {
 class ReconciliationReport extends React.Component {
   constructor () {
     super();
-    this.reload = this.reload.bind(this);
     this.navigateBack = this.navigateBack.bind(this);
+    this.handleCardClick = this.handleCardClick.bind(this);
+    this.state = {
+      activeIdx: 0
+    };
   }
 
   componentDidMount () {
-    const { reconciliationReportName } = this.props.match.params;
-    const immediate = !this.props.reconciliationReports.map[reconciliationReportName];
-    this.reload(immediate);
-  }
-
-  componentWillUnmount () {
-    if (this.cancelInterval) { this.cancelInterval(); }
-  }
-
-  reload (immediate) {
-    const { reconciliationReportName } = this.props.match.params;
-    const { dispatch } = this.props;
-    if (this.cancelInterval) { this.cancelInterval(); }
-    this.cancelInterval = interval(() => dispatch(getReconciliationReport(reconciliationReportName)), updateInterval, immediate);
+    const { dispatch, match } = this.props;
+    const { reconciliationReportName } = match.params;
+    dispatch(getReconciliationReport(reconciliationReportName));
```

Comment:
I also know this predates this PR, but why do we dispatch the report every time we display it?  we're also storing it in map and fetching it every time it seems...

Reply:
My guess would be to ensure we get the freshest version of of the report every time we get to the page. Originally had this wrapped in an if statement so that it would dispatch only if the report was not in the map, but I thought that might have been causing the issue Lauren was seeing. Can add it back though

Reply:
So I was unfamiliar with this, I don't know if the reports change once they're available.  If not we should add it back, if so, this is fine.  let's ask Lauren. @laurenfrederick

Reply:
After discussing with Lauren on scrum, these will never change, so I put this back in the conditional